src/renderers/native/ReactNativeFiber.js

```diff
@@ -416,4 +417,11 @@ const ReactNative = {
 
 };
 
+if (typeof injectInternals === 'function') {
```

Comment:
I still think this should be unconditional, here and move the conditional into the other module, so that it can be inlined AOT and we don't have to think about this possibly being null.